spec/controllers/static_pages_controller_spec.rb

```diff
@@ -14,9 +14,9 @@
   describe 'Get #index' do
     it 'has preview events' do
       VCR.use_cassette('8th_light_team') do
-        MockEventsHelper.mock_events(:this_week, 4)
+        MockEventsHelper.mock_events(:upcoming, 4)
         get :index
-        expect(controller.instance_variable_get(:@preview_events).length).to eq(4)
+        expect(controller.instance_variable_get(:@preview_events).length).to eq(3)
```

Comment:
This test will fail randomly due to mixing the `:upcoming` and `:this_week` strategies.  Take a look at the strategies and you will see why this will fail randomly.